apps/system/test/unit/lockscreen/widgets/clock/lockscreen_clock_widget_test.js

```diff
@@ -41,12 +40,12 @@ suite('LockScreenClockWidget > ', function() {
       }}
     };
     var method = LockScreenClockWidget.prototype.updateClock;
+    var now = new Date();
     method.call(mockThis);
-    assert.equal(navigator.mozL10n.dummyLocaleFormat(),
+    assert.equal(mockThis.timeFormatter.format(now),
```

Comment:
Are you sure the test was passed? Since I see you remove the mocked `Date.now`, so it would not print the dummy text but the real formatted date & time, which shall not match the `mockThis.resources.elements.date.textContent`.